zmsadmin/js/page/availabilityDay/index.js

```
@@ -271,14 +319,14 @@ class AvailabilityPage extends Component {
                 selectedAvailability: null
             })
         }
-        
+
     }
 
     editExclusionAvailability(availability, startDate, endDate, description, kind) {
         (startDate) ? availability.startDate = startDate : null;
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)